SparkConnector/setup.py

```diff
@@ -63,7 +63,7 @@
     cmdclass= cmdclass,
     packages=setuptools.find_packages(),
     install_requires=[
-        "jupyterlab~=3.0",
+        "jupyterlab>=4.0.0,<5",
```

Comment:
Here JupyterLab is required to install and to build?

Reply:
I'm not understanding the question.

Reply:
Ah sorry, yes, you need lab to build things as well. Don't remember why, but that is part of their cookiecutter recipe.